lib/matplotlib/backends/backend_tkagg.py

```diff
@@ -340,6 +342,11 @@ def motion_notify_event(self, event):
         y = self.figure.bbox.height - event.y
         FigureCanvasBase.motion_notify_event(self, x, y, guiEvent=event)
 
+    def enter_notify_event(self, event):
```

Comment:
This is probably better named something else (as it is new)

Reply:
Could but breaks naming scheme used in the class.